Stripe/STPAPIPostRequest.h

```diff
@@ -12,7 +12,7 @@
 
 @interface STPAPIPostRequest<__covariant ResponseType:id<STPAPIResponseDecodable>> : NSObject
 
-typedef void(^STPAPIPostResponseBlock)(ResponseType object, NSError *error);
+typedef void(^STPAPIPostResponseBlock)(ResponseType object, NSURLResponse *response, NSError *error);
```

Comment:
Can you just make this be an NSHTTPURLResponse, cast it internally, and avoid having to cast it elsewhere?